### DISCUSSION TITLE
在编辑表单中，无法获取到正在编辑的实例的值

### DISCUSSION THREAD

**Discussion by qreal829475** [alibaba/formily#2599] [Q&A 2.x] [answered]
https://codesandbox.io/s/formily-json-demo-2whlf?file=/App.tsx

在编辑数组的demo中，无法拿到popover中的项的值
我是在原来的编辑详情demo中，最外层增加了FormLayout作为表单的布局。

在255行中的console，我打出来的self中，$self.query('.name').value()失败

如果去掉schema中最外层的FormLayout就可以正常使用？ 求解答，这个问题已经困惑得走不动了

**Comment by janryWang** [comment, marked as answer]
用dependencies就行了呀，干嘛还要单独去查

**Reply by qreal829475** [reply]
但是我看$self中说的是拿到当前的实例，但是为什么包装一层以后就拿不到了，感觉很困惑。
我去试试dependencies
